content/300-guides/050-database/870-using-prisma-with-mongodb.mdx

```diff
@@ -0,0 +1,307 @@
+---
+title: 'Using Prisma with MongoDB'
+metaTitle: 'Using Prisma with MongoDB'
+metaDescription: 'Guide to using Prisma with MongoDB'
+tocDepth: 2
+toc: true
+---
+
+<TopBlock>
+
+This guide discusses the concepts behind using Prisma and MongoDB, explains the commonalities and differences between MongoDB and other database providers, and leads you through the process for configuring your application to integrate with MongoDB.
+
+</TopBlock>
+
+## What is MongoDB?
+
+MongoDB is a NoSQL database that stores data in [BSON](https://bsonspec.org/) format, a JSON-like document format designed for storing data in key-value pairs.
+
+MongoDB is commonly used in JavaScript application development because the document model maps easily to objects in application code, and there is built in support for high availability and horizontal scaling.
+
+MongoDB stores data in collections that do not need a schema to be defined in advance, as you would need to do with tables in a relational database. The structure of each collection can also be changed over time. This flexibility can allow rapid iteration of your data model, but it does mean that there are a number of differences when using Prisma to work with your MongoDB database.
+
```

Comment:
I realise this is 3 paragraphs starting with 'MongoDB' in a row, which doesn't look great. Will sort this out once the content's been settled.

Reply:
Made it somewhat better.